docs/source/install/kernel_install.rst

```diff
@@ -53,3 +53,33 @@ For example, using conda environments:
 The ``--name`` value is used by Jupyter internally. These commands will overwrite
 any existing kernel with the same name. ``--display-name`` is what you see in
 the notebook menus.
+
+Using virtualenv you can add your new environments by linking to specific instance of your Python:
+
+.. sourcecode:: bash
+
+    python -m ipykernel install --prefix=/path/to/new_env --name python_new_env
+  
+Note that this command will create new configuration for kernel in one of it's prefered location (see ``jupyter --paths`` command for more details):
+
+* system-wide (e.g. /usr/local/share),
+* in Jupyter's env (sys.prefix/share),
+* per-user (~/.local/share or ~/Library/share)
+
+You can also create configuration in temporary location and install it in Jupyter (copy configuration files) with:
+
+.. sourcecode:: bash
+
+    ipython kernel install --prefix /tmp
+    jupyter kernelspec install /tmp/share/jupyter/kernels/python3
+
+**Deprecated:** You can also create configuration for new environment by new configuration's file in `~/.ipython/kernels/my_env/kernel.json` with following content::
```

Comment:
Let's remove the discussion of the deprecated case. We don't want anyone to be putting kernels here, and adding it to the docs only makes it more likely.